msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -16,6 +16,13 @@ A proactive message is any message sent by a bot that isn't in response to a req
 * Notifications
 * Scheduled messages
 
+> [!IMPORTANT]
+> Currently, bots are available in Government Community Cloud (GCC) and GCC-High but not in Department of Defense (DOD).
+>
+> For proactive messages the bots should use the following end points for government cloud environments:
```

Comment:
Is this different from the service URLs that they get in the activity?

This can easily be read as meaning that one needs to **replace** the service URL received in the activity with these URLs. Is that what we mean?

Reply:
+1, we should add more clarity here. What does this URL mean and how should it be consumed?